week7/7.3_homework.ipynb

```diff
@@ -2,76 +2,109 @@
-    "- __(0.5 pts)__ No one said l2 loss is perfect. Implement Huber or MAE loss for DRQN and/or A2C critic and compare it's performance on the game of your choice (pass proper `loss_function` to `get_elementwise_objective()`) .\n",
-    "- __(1 pts)__ See the very end of this notebook. Some of the games (right side) benefit a lot from additional LSTM memory. But others (left side) do not. That is interesting. Pick up one or several games from the left side and try to figure out why A2C performance decreases when adding LSTM to feadforward architecture?\n",
+    "__Submisson__: To ease mutual pain, please submit\n",
+    "- Some kind of readable report with links to your evaluations, gym uploads, investigation results, etc.\n",
+    "- Explicitly state that you took on a bonus task and where to find it [to make sure it is found and graded].\n",
     "\n",
+    "__Grading__: The main purpose (and source of points) for this notebook is your investigation, not squeezing out average rewards from random environments. \n",
     "\n",
+    "Getting near/above state of the art performance on one particular game will earn you some bonus points, but you can get much more by digging deeper into what makes the algorithms tick and how they compare to one another.\n",
     "\n",
-    "__NB__: This homework will be graded mainly according to the amount of effort put in investigation, not on the final performance you could squeeze out of the particular method on the particular game. However if you feel more excited  about tuning hyperparameters and find the questions above boring you may skip question answering and proceed to the risky bonus part below.  \n",
+    "Okay, now brace yourselves, here comes an assignment!\n",
     "\n",
-    "__Tips__: \n",
-    "- when you crop your image in atari games you should also appopriately reduce filter sizes and amount of filters / size of your Q- / policy network\n",
-    "- don't forget that PG methods need regularizaiton for proper training because of inflated gradient variance when policy becomes deterministic."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "deletable": true,
-    "editable": true
-   },
-   "source": [
-    "# II - Bonus \n",
+    "#### 7.1 rock-paper-sarsa\n",
+    "Go to [the first notebook](https://github.com/yandexdataschool/Practical_RL/blob/master/week7/7.1_rock_paper_scissors.ipynb) and implement basic recurrent SARSA.\n",
+    "\n",
+    "_note: there's a bonus task at the end of this notebook_\n",
+    "\n",
+    "#### 7.2 Deep kung-fu (3 pts)\n",
+    "\n",
+    "Implement and train recurrent actor-critic on `KungFuMaster-v0` env in the second notebook. Try to get a score of >=20k.\n",
+    "\n",
+    "Please __upload your algorithm to [gym leaderboard]__(https://gym.openai.com/envs/KungFuMaster-v0) and submit the link to your eval!\n",
+    "\n",
+    "\n",
+    "#### 7.3 new horizons (7 pts)\n",
```

Comment:
The task worth 7pts is making them both work "okay" on chosen env and comparing performance. Clarified.